metaswitch/common/pdlogs.py

```diff
@@ -48,6 +49,7 @@ class PDLog(object):
     CL_CONFIG_MGR_ID = 9000
     # Ranges 10000 to 11999 are reserved
     CL_PYTHON_COMMON_ID = 12000
+    CL_CREST_ID = 13000
```

Comment:
Can you update https://github.com/Metaswitch/cpp-common/blob/master/include/pdlog.h#L85 with this? Can you also add a comment here as well to note that these should be kept in sync?